woocommerce/class-sv-wc-plugin-compatibility.php

```diff
@@ -491,7 +491,17 @@ protected static function is_wc_subscriptions_version( $version ) {
 		return $version === $subscriptions_version || ( $subscriptions_version && version_compare( $version, $subscriptions_version, '=' ) );
 	}
 
+	/**
+	 * Whether the HPOS is enabled.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return bool
+	 */
+	public static function is_hpos_enabled() {
 
+		return class_exists( $custom_order_controller = \Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController::class ) && 'yes' === get_option( $custom_order_controller::CUSTOM_ORDERS_TABLE_USAGE_ENABLED_OPTION );
```

Comment:
```suggestion
		$custom_orders_table_controller = \Automattic\WooCommerce\Internal\DataStores\Orders\CustomOrdersTableController::class;
		
		return class_exists( $custom_orders_table_controller )
			&& 'yes' === get_option( $custom_order_controller::CUSTOM_ORDERS_TABLE_USAGE_ENABLED_OPTION );
```

This is a rather long line, maybe can be shortened for readability